apps/api/.env.example

```diff
@@ -11,4 +11,11 @@ COINGECKO_API_KEY='' # CoinGecko API key
 COINGECKO_BASE_URL='' # CoinGecko API base URL for the API version you are using
 COINGECKO_API_TYPE='' # CoinGecko API Type: 'demo' or 'pro'
 
-CACHE_TTL=60 # Cache TTL in seconds
\ No newline at end of file
+CACHE_TTL=60 # Cache TTL in seconds
```

Comment:
i would add more details on this, is for pricing or endpoints ?

Reply:
i didn't split here since user config cache for API is left for future improvement and separete cache for each provider/service is also an enhancement (since now we instantiate a single shared cache)